sql/share/errmsg-utf8.txt

```diff
@@ -2800,7 +2802,7 @@ ER_STACK_OVERRUN
         rus "Стек потоков переполнен:  использовано: %ld из %ld стека.  Применяйте 'mariadbd --thread_stack=#' для указания большего размера стека, если необходимо"
         serbian "Prepisivanje thread stack-a:  Upotrebljeno: %ld od %ld stack memorije.  Upotrebite 'mariadbd --thread_stack=#' da navedete veći stack ako je potrebno"
         slo "Pretečenie zásobníku vlákna:  použité: %ld z %ld.  Použite 'mariadbd --thread_stack=#' k zadaniu väčšieho zásobníka"
-        spa "Sobrecarga de la pila de thread:  Usada: %ld de una %ld pila.  Use 'mariadbd --thread_stack=#' para especificar una mayor pila si necesario"
+        spa "Desbordamiento de la pila de hilos: Usado: %ld de una pila de %ld. Considera el incrementar la variable de sistema thread_stack"
```

Comment:
I am wondering if `(threads)` should be used also...

Reply:
I agree. It should be ... "de la pila de hilos (threads):" instead of "de la pila de hilos"

Reply:
@jesusaplsoft would you consider making those changes and update the PR?